gandalf.js

```diff
@@ -11,7 +11,8 @@
  */
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!